README.md

```diff
@@ -16,10 +16,42 @@ For this, we use an ADR-inspired process.
 
 To make a meaningful contribution:
 
-1. Create an ADR proposal in the [decisions/log](decisions/log/) with status `In Progress`
-    1. If possible, already create the changes to the technical specification text as well (see above)
-2. Create a Pull Request with the ADR proposal and optionally the specification changes
-3. Once there is consensus on the ADR proposal, the ADR proposal is merged into the `main` branch
+## ADR Submission, Review, and Acceptance Process
+Architecture decisions are submitted via creating a ``Markdown``
+
+- ADRs are numbered sequentially and monotonically. Numbers are not reused.
+- Format of the ADR: Please see an [existing ADR](https://github.com/wbcsd/data-exchange-protocol/blob/adr-0021-auth-conformance-issues/decisions/log/0015-pagination.md) for the format
```

Comment:
Is there a reason why we don't share a "template" for the ADR? Many times in the past few months I have been asked for the ADR template - folks like to copy and paste it, and then fill it out. Currently it is here but think this should be in github and we would eliminate it in .docx https://wbcsd.sharepoint.com/:w:/s/ClimateEnergy/Efozt_RoKPFFsAvQ97B-gKIB1XeC-er3jxuFxTxiJd9-ZA?e=wH9vq2 Would be good to save them time, and also standardized the format of the ADR. Would also recommend including an example in addition to the template.

Reply:
I added a (minimal) template as part of this PR: [decisions/template/adr-template.md](https://github.com/wbcsd/data-exchange-protocol/pull/17/files#diff-ef2d552d6c024a6a955e8e851bcb8a09cf4d3f9bcc2fa93582e66ca86d9fd5c3). 
It is not as comprehensive as the .docx document, however.